fetch-words.js

```diff
@@ -4,9 +4,53 @@ const got = require('got');
 
 const url = 'https://raw.githubusercontent.com/atebits/Words/master/Words/en.txt';
 
+const badWords = require('badwords-list').array;
+const leoProfanity = require('leo-profanity');
+
+leoProfanity.loadDictionary('en');
+
+const badWordsSet = new Set(badWords);
+const badWordsExclusions = new Set(['ass', 'asses', 'anal', 'cum', 'tit', 'bum', 'hore', 'sex', 'cipa', 'butt', 'cock', 'homo', 'eros', 'cox', 'semen', 'damn', 'jap', 'lust', 'hell', 'pron', 'crap']);
+const badWordsWithExclusions = badWords.filter(word => !badWordsExclusions.has(word));
+
+const filters = [
+	{
+		name: 'leo-profanity',
+		fn: word => !leoProfanity.check(word)
+	},
+	{
+		name: 'badwords-list exact',
+		fn: word => !badWordsSet.has(word)
+	},
+	{
+		name: 'badwords-list with exclusions substring',
+		fn: word => {
+			for (const badWord of badWordsWithExclusions) {
+				if (word.indexOf(badWord) !== -1) {
```

Comment:
Use `.includes()` instead. Same in the tests.

Reply:
Done, but I think you're really rushing that `includes` stuff. Not that it was really important, but using `includes` instead of `indexOf` increases run time of whole build by 15% in this case. (`indexOf` is core function and is well optimized everywhere, unlike `includes`, which, as far as I know, is slow in every major implementation.)

Reply:
Doesn't matter in this case. The script is only run maybe once a year.